.github/workflows/default_unittest.yml

```diff
@@ -0,0 +1,33 @@
+# This workflow will build a golang project
+# For more information see: https://docs.github.com/en/actions/automating-builds-and-tests/building-and-testing-go
+
+name:  Unit Tests
+
+on:
+  workflow_dispatch:
```

Comment:
is this file really needed? the complete test also can be triggered manually

Reply:
I thought it depends on the size of integration tests. If running scripts `tests/run_tests.sh` takes much longer than whole unit tests. But after hearing what you said, I think it's better run it directly on the local machine when I want to test.